src/com/facebook/buck/jvm/java/intellij/ij-module.st

```diff
@@ -49,7 +49,14 @@
         %if(generatedFolder.isAndroidResources)% type="java-resource" %endif%/>
     </content>
 }%
+
+%if(!androidFacet.enabled && java_sdk)%
```

Comment:
just like with IjProjectWriter, the templates are only tested through integration tests, so it's more convenient to have any logic live at higher abstraction levels. Here I suggest adding a Jdk type on the module, if it's null, print 'inheritedJdk', otherwise print it's .name and .type:

```
%if(jdk)%
  <orderEntry type="jdk" jdkName="%jdk.name%" jdkType="%jdk.type%" />
%else%
 <orderEntry type="inheritedJdk" />
%endif%
```